libpebble3/src/commonMain/kotlin/io/rebble/libpebblecommon/packets/blobdb/TimelineIcon.kt

```diff
@@ -102,7 +102,21 @@ enum class TimelineIcon(val id: UInt, val code: String) {
     AlarmClock(13u, "system://images/ALARM_CLOCK"),
     HeavySnow(53u, "system://images/HEAVY_SNOW"),
     DinnerReservation(27u, "system://images/DINNER_RESERVATION"),
-    NotificationKakaoTalk(79u, "system://images/NOTIFICATION_KAKAOTALK");
+    NotificationKakaoTalk(79u, "system://images/NOTIFICATION_KAKAOTALK"),
+    NotificationBeeper(120u, "system://images/NOTIFICATION_BEEPER"),
+    NotificationDiscord(121u, "system://images/NOTIFICATION_DISCORD"),
+    NotificationBluesky(122u, "system://images/NOTIFICATION_BLUESKY"),
+    NotificationDuolingo(123u, "system://images/NOTIFICATION_DUOLINGO"),
+    NotificationElement(124u, "system://images/NOTIFICATION_ELEMENT"),
+    NotificationGoogleChat(125u, "system://images/NOTIFICATION_GOOGLE_CHAT"),
```

Comment:
why not replace GChat over Hangouts? hangouts no longer exists, and GChat grew out of it